src/frontend/components/Chat/SharedChatComponents/ChatInputBar/index.spec.tsx

```diff
@@ -0,0 +1,52 @@
+import { act, render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import { ChatInputBar } from '.';
+import { wrapInIntlProvider } from '../../../../utils/tests/intl';
+import { converse } from '../../../../utils/window';
+
+jest.mock('../../../../utils/window', () => ({
+  converse: {
+    sendMessageWithConverse: jest.fn(),
+  },
+}));
+
+describe('<ChatInputBar />', () => {
+  it('emulates a click on the send button when input is empty', async () => {
```

Comment:
```suggestion
  it('does send message if the send button is clicked when input is empty', async () => {
```

Reply:
`doesn't`, he expects that `sendMessageWithConverse` is not called.

Reply:
done